sbi/inference/snpe/snpe_c.py

```diff
@@ -107,18 +106,7 @@ def __call__(
         Return posterior $p(\theta|x)$ after inference (possibly over several rounds).
 
         Args:
-            num_rounds: Number of rounds to run. Each round consists of a simulation and
-                training phase. `num_rounds=1` leads to a posterior $p(\theta|x)$ valid
-                for _any_ $x$ (amortized), but requires many simulations.
-                Alternatively, with `num_rounds>1` the inference returns a posterior
-                $p(\theta|x_o)$ focused on a specific observation `x_o`, potentially
-                requiring less simulations.
-            num_simulations_per_round: Number of simulator calls per round.
-            x_o: An observation that is only required when doing inference
-                over multiple rounds. After the first round, `x_o` is used to guide the
-                sampling so that the simulator is run with parameters that are likely
-                for that `x_o`, i.e. they are sampled from the posterior obtained in the
-                previous round $p(\theta|x_o)$.
+            num_simulations: Number of simulator calls per round.
```

Comment:
Docstring could be improved, there is no explicit concept of rounds at the level of the `__call__` keywords anymore